Atomic/syscontainers.py

```diff
@@ -398,12 +434,18 @@ def _get_ostree_repo(self):
 
     def update_system_container(self, name):
         self.args.display = False
+        self.args.remote = None
 
         repo = self._get_ostree_repo()
         if not repo:
             raise ValueError("Cannot find a configured OSTree repo")
 
         path = os.path.join(self._get_system_checkout_path(), name)
+        with open(os.path.join(path, "info"), 'r') as info_file:
+            info = json.loads(info_file.read())
+            self.args.remote = info['remote']
+            if self.args.remote:
+                util.write_out("Updating a container with a remote rootfs. Only changes to config will be applied.")
```

Comment:
Hm so one of two things could happen:

1. the user runs --update on an existing container which HAS a rootfs, and that rootfs is also in use by other containers as a remote rootfs. Then the rootfs gets updated, BUT the remote ones still point to the old (since it resolved the true path instead of the symlink). i.e. the container goes to container.1 but the remotes still use container.0

2. the user runs update DIRECTLY on a container with a remote rootfs, which cannot modify the remote rootfs, but can update the config.

In hindsight this is probably not a good way to do it. I think what I will have it do is: if the original container is updated, all the remote automatically use the new rootfs (by just saving the symlink) unless the user specifies not to.